lib/ansible/modules/monitoring/zabbix/zabbix_host_events_info.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) stephane.travassac@fr.clara.net
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+RETURN = '''
+---
+triggers_ok:
+    description: Host Zabbix Triggers in OK state
+    returned: On success
+    type: complex
+    contains:
+            - comments: Additional description of the trigger (https://www.zabbix.com/documentation/3.0/manual/api/reference/trigger/object)
+              description: Name of the trigger
+              error: Error text if there have been any problems when updating the state of the trigger
+              expression: Reduced trigger expression
+              flags: Origin of the trigger
+              lastchange: Time when the trigger last changed its state (timestamp)
+              priority: Severity of the trigger
+              state: State of the trigger
+              status: Whether the trigger is enabled or disabled
+              templateid: ID of the parent template trigger
+              triggerid: ID of the trigger
+              type: Whether the trigger can generate multiple problem events
+              url: URL associated with the trigger
+              value: Whether the trigger is in OK or problem state
+triggers_problem:
+    description: Host Zabbix Triggers in problem state (https://www.zabbix.com/documentation/3.0/manual/api/reference/trigger/object & https://www.zabbix.com/documentation/3.0/manual/api/reference/event/object)
```

Comment:
Ideally this should be covered in documentation of module so links are not necessary, but 
```suggestion
    description: Host Zabbix Triggers in problem state. See trigger and event objects in API documentation of your zabbix version for more
```